src/main/java/com/photolocos/enterprise/dao/LocationSQLDAO.java

```diff
@@ -42,6 +42,11 @@ public LocationDTO fetchByArea(String state, String city) {
 
         return byStateAndCity;
     }
+    
+    @Override
+    public void delete(int id) {
+        locationRepository.deleteByID(id);
+    }
```

Comment:
Is this used anywhere?  I don't see any references to it.

If it is not used anywhere, it is unused code, which is clutter, and increases technical debt.  Only features that are complete and meet the team's Definition of Done should be merged to master.